server/webapp/WEB-INF/rails/webpack/models/materials/config_file_list.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Stream} from "mithril/stream";
+import stream from "mithril/stream";
```

Comment:
If you rebase, you probably won't need two imports for `Stream`. Generally, this will do from now on (note the titlecase):

```typescript
import Stream from "mithril/stream";
```